### PR TITLE
Feature ia grpc client

### PR DESCRIPTION
**Pull request by Decimusprimus**

### CODE REVIEW

File: Backend/Examinations/Service/Interfaces/ITreatmentService.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using Backend.Examinations.Model;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Backend.Examinations.Service.Interfaces
+{
+    public interface ITreatmentService
+    {
+        public Treatment CreateTreatment(Treatment treatment);
```
Comment (oakenknight) [inline Backend/Examinations/Service/Interfaces/ITreatmentService.cs]: remove public

File: PharmacyIntegration/gRPC/GrpcClient.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        public GrpcClient() { }
+
+        public async Task<string> CheckForMedication(string name)
+        {
+            var channel = new Channel("localhost:6565", ChannelCredentials.Insecure);
+            var client = new CheckForMedication.CheckForMedicationClient(channel);
+
+            MessageResponseProto response = await client.checkAsync(new MessageProto() { Message = name });
+            Console.WriteLine(response.Response + " odgovor");
```
Comment (oakenknight) [inline PharmacyIntegration/gRPC/GrpcClient.cs]: odgovor -> response

Comment (oakenknight) [reply PharmacyIntegration/gRPC/GrpcClient.cs]: or remove completly

**Review by oakenknight — CHANGES_REQUESTED** [review]
correct interface and remove console.writeline

File: PharmacyIntegration/vueclient/src/views/Medication.vue [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                })
+        },
+        sendPrescription: function (prescription) {
+            this.axios.post("http://localhost:50202/api/Prescription/", prescription)
+                .then(response => {
+                    //This is for sftp
+                    /*this.axios.get("http://localhost:50202/api/Sftp")
+                        .then(response => {
+                            console.log(response.data);
+                        })*/
```
Comment (st3fan7) [inline PharmacyIntegration/vueclient/src/views/Medication.vue:114]: Remove comments

**Review by st3fan7 — CHANGES_REQUESTED** [review]

**Review by oakenknight — APPROVED** [review]

**Review by st3fan7 — APPROVED** [review]

**Pull request merged**